library/a-star/src/main/java/astar/Spot.java

```diff
@@ -5,26 +5,28 @@
 
 public class Spot {
 
+    private int x;
+    private int y;
+    public boolean wall;
+
     private double f;
     private double g;
     private double h;
-    private int x;
-    private int y;
     private List<Spot> neighbors;
```

Comment:
code convention: m prefix for member variables, also what are f g h (I am assuming x,y are coordinates). please use more meaningful variable names